sign-page/src/App.css

```diff
@@ -0,0 +1,96 @@
+.head{
+   padding-right: 180px;
+    line-height: 63.68px;
+    
+}
+.form{
+    text-align: center;
+    width: 322px;
+    height: 340px;
+  
+}
+h3 {
+    font-weight: normal;
+    margin-left: 15px;
+}
+#email{
+    width:288px;
+    height: 33px;
+    display: flex;
+    margin-top: 8px;
+    margin-left: 32px;
+    border-radius: 4px;
+   border: Solid rgba(112, 144, 176, 0.4);
+    
+}
+#password{
+    width: 288px;
+    height: 33px;
+    display: flex;
+    margin-top: 8px;
+    margin-left: 32px;
+    border-radius: 4px;
+    border: Solid rgba(112, 144, 176, 0.4);
+}
+
+.word{
+    font-size: 12px;
+    margin-right: 230px;
+}
+.mobile {
+    font-size: 12px;
+    margin-right: 132px;
+}
+.number{
+    margin-top: 13px;
+}
+
+.remember{
+    font-size: 12px;
+    margin-left: 8px;
+}
+body{
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    height: 100vh;
+}
+button{
+    margin-top: 10px;
+    width:287px;
+    height: 37px;
+    background: #00508F;
+    border-radius: 6px;
+    border: none;
+    color: white;
+    font-size: 12px;
+    font-family: 'Open Sans', sans-serif;
+}
+img{
+    width:170px;
+}
+::placeholder {
+    padding-left: 5px;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 10px;
+    
```

Comment:
Don't set padding for placeholder, instead set the padding for the entire input itself.